pkg/operator/sync.go

```diff
@@ -140,20 +140,41 @@ type syncError struct {
 }
 
 func (optr *Operator) syncAll(syncFuncs []syncFunc) error {
-	if err := optr.syncProgressingStatus(); err != nil {
-		return fmt.Errorf("error syncing progressing status: %w", err)
+
+	co, err := optr.fetchClusterOperator()
+	if err != nil {
+		return err
+	}
+	if co == nil {
```

Comment:
Curious, when can this be nil? Just very early on in the install cycle?

Reply:
good catch! I'll take this bit out. `fetchClusterOperator()` will create a CO if doesn't exist, so we could probably take this nil check out.